<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_options">
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>650</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Options and Information</string>
      </property>
      <widget class="QDoubleSpinBox" name="doubleSpinBx_scaleFactor">
       <property name="geometry">
        <rect>
         <x>273</x>
         <y>49</y>
         <width>59</width>
         <height>24</height>
        </rect>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.200000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QPushButton" name="pushBtn_ZoomOut">
       <property name="geometry">
        <rect>
         <x>383</x>
         <y>49</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Zoom Out</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_AvailableImages">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>80</y>
         <width>109</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Available Images</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_priorityImages">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>80</y>
         <width>96</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Priority Images</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushBtn_ZoomIn">
       <property name="geometry">
        <rect>
         <x>142</x>
         <y>49</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Zoom In</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chkBx_FitToWindow">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>24</y>
         <width>98</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Fit To Window</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushBtn_remove">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>260</y>
         <width>91</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushBtn_add">
       <property name="geometry">
        <rect>
         <x>279</x>
         <y>230</y>
         <width>91</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>98</y>
         <width>256</width>
         <height>611</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QListView" name="listView_availImages"/>
       <widget class="QWidget" name="widget" native="true">
        <widget class="QLabel" name="label_ImageInformation">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>0</y>
           <width>117</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
           <underline>false</underline>
          </font>
         </property>
         <property name="text">
          <string>Image Information</string>
         </property>
        </widget>
        <widget class="QTextBrowser" name="textBrowser_ImageInfo">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>21</y>
           <width>256</width>
           <height>231</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_2">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>100</y>
         <width>256</width>
         <height>611</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QListView" name="listView_priorityImages"/>
       <widget class="QWidget" name="widget_2" native="true">
        <widget class="QLabel" name="label_imageInfo_priority">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>0</y>
           <width>117</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Image Information</string>
         </property>
        </widget>
        <widget class="QTextBrowser" name="textBrowser_imageInfo_priority">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>261</width>
           <height>192</height>
          </rect>
         </property>
        </widget>
        <zorder>label_imageInfo_priority</zorder>
        <zorder>textBrowser_imageInfo_priority</zorder>
        <zorder>listView_priorityImages</zorder>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>922</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Image"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Image">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
